cmd/controller/app/options/options.go

```diff
@@ -183,6 +191,8 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"will be attempted.")
 	fs.StringVar(&s.Kubeconfig, "kubeconfig", defaultKubeconfig, ""+
 		"Paths to a kubeconfig. Only required if out-of-cluster.")
+	fs.Float32Var(&s.KubernetesAPIQPS, "kubernetes-api-qps", defaultKubernetesAPIQPS, "indicates the maximum QPS requests to the Kubernetes master")
```

Comment:
can we avoid the use of the term `master` in both of these?

Also, perhaps reword to `the maximum queries-per-second of requests sent to the Kubernetes apiserver`?

Reply:
Agreed, was doubting about it as it is a copy paste from the k8s docs wasn't too sure if apiserver was correct